.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+Linked PRs: (links to corresponding PRs, optional)
+
+## Changes
+- (changes here)
+
+## TODOs:
+- [ ] +1
+- [ ] Updated README
+- [ ] Updated CHANGELOG
+- [ ] (Other applicable TODOs)
+- [ ] (Feedback from reviewers)
+
+## How To Test
+- (necessary config changes)
+- (necessary corresponding PRs)
+- (how to access the new / changed functionality -- fixtures, URLs)
```

Comment:
I feel this **How to Test** section doesn't sound related to the actual code-style repo. I'm not sure "How to Test" is even needed for the code-style repo.